docker/tests/containerTestCommon.psm1

```diff
@@ -243,6 +243,7 @@ function Test-PSPackage
         $buildArgs += "--build-arg","$versionStubName=$versionStubValue"
         $buildArgs += "--build-arg","$testlistStubName=$testlistStubValue"
         $buildArgs += "--build-arg","$packageLocationStubName=$packageLocationStubValue"
+        $buildArgs += "--no-cache"
```

Comment:
Please don't do this.  This will make the other tests very slow.  Please fix your docker files, or make this an option, that only your tests use.

Reply:
after talking to @anmenaga  offline this looks fine.